samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -394,46 +418,46 @@ static void modem_connect(void)
 	do {
 		LOG_INF("Connecting to LTE network.");
 		LOG_INF("This may take several minutes.");
-		ui_led_set_pattern(UI_LTE_CONNECTING);
 
 		ret = lte_lc_connect();
 		if (ret < 0) {
-			LOG_WRN("Failed to establish LTE connection (%d).",
-				ret);
+			LOG_WRN("Failed to establish LTE connection (%d).", ret);
 			LOG_WRN("Will retry in a minute.");
 			lte_lc_offline();
 			k_sleep(K_SECONDS(60));
 		} else {
 			LOG_INF("Connected to LTE network");
-			ui_led_set_pattern(UI_LTE_CONNECTED);
 		}
 	} while (ret < 0);
 }
 
 void main(void)
 {
 	uint32_t flags = IS_ENABLED(CONFIG_LWM2M_RD_CLIENT_SUPPORT_BOOTSTRAP) ?
-				    LWM2M_RD_CLIENT_FLAG_BOOTSTRAP : 0;
+				       LWM2M_RD_CLIENT_FLAG_BOOTSTRAP :
+				       0;
 	int ret;
 
 	LOG_INF(APP_BANNER);
 
 	k_sem_init(&lwm2m_restart, 0, 1);
 
-	ui_init(ui_evt_handler);
+	ret = event_manager_init();
+	if (ret) {
+		LOG_ERR("Unable to init event manager (%d)", ret);
+		return;
+	}
 
-#if defined(CONFIG_LWM2M_CLIENT_UTILS_FIRMWARE_UPDATE_OBJ_SUPPORT)
```

Comment:
I would say this should be put back unless there is some good reason behind this change.

Reply:
Fixed